language/stdlib/transaction_scripts/add_validator.move

```diff
@@ -1,12 +1,21 @@
 script {
     use 0x1::LibraSystem;
+    use 0x1::SlidingNonce;
+    use 0x1::ValidatorConfig;
 
     /// Add `new_validator` to the validator set.
     /// Fails if the `new_validator` address is already in the validator set
     /// or does not have a `ValidatorConfig` resource stored at the address.
     /// Emits a NewEpochEvent.
     /// TODO(valerini): rename to add_validator_and_reconfigure?
```

Comment:
Might be a good opportunity to do this since we're making a breaking change to the script (same with scripts below).